boot/bootutil/src/loader.c

```diff
@@ -3185,7 +3205,8 @@ context_boot_go(struct boot_loader_state *state, struct boot_rsp *rsp)
     while (true) {
 #endif
         FIH_CALL(boot_load_and_validate_images, fih_rc, state);
-        if (fih_not_eq(fih_rc, FIH_SUCCESS)) {
+        if (FIH_NOT_EQ(fih_rc, FIH_SUCCESS)) {
+            fih_rc = FIH_FAILURE;
```

Comment:
`FIH_SET`?

Reply:
seems to be not really needed here (according to fault injection simulations)